include/wil/stl.h

```diff
@@ -19,6 +19,7 @@
 #include <string>
 #include <vector>
 #include <utility>
+#include <TraceLoggingProvider.h>

```

Comment:
Technically this is illegally reaching into internal implementation details of TraceLoggingProvider.h.

Probably the right thing to do is:

1. Modify TraceLoggingProvider.h to expose enough "supported" machinery to support some C++ constructs.
2. Modify WIL to use the supported mechanism if present, or to poly-fill the machinery if not present. (The poly-fill will  only be used on old versions of TraceLoggingProvider.h, so future changes to TraceLoggingProvider.h internals will not break the poly-fill.